frontend/src/app/utils/dates/timeline.helper.ts

```diff
@@ -0,0 +1,155 @@
+import { addDays, addHours, addMonths, addWeeks, addYears, daysInWeek, differenceInHours, differenceInYears, endOfDay, endOfMonth, endOfWeek, endOfYear, startOfDay, startOfMonth, startOfWeek, startOfYear } from 'date-fns';
+
```

Comment:
helper is a bad naming. find a name that reflects what this file contains.

Reply:
Fixed